docs/test-pages/TCD-008.adoc

```diff
@@ -0,0 +1,36 @@
+= TCD-008
+
+== Purpose
+
+Verifying the primitive types and their behaviour. This case is verifying the optional primitive fields without adding any values to them.
```

Comment:
Is this an SDK test, or IDE plugin? Test targets should be defined in every testing document IMO.

Reply:
It is a reverse engineered test case. It was a test to me how long to do this. By the way it could be implemented on the JSL level, and SDK level too. Or any level.